.gitmodules

```diff
@@ -33,3 +33,7 @@
 	path = libraries/standard/backoffAlgorithm
 	branch = main
 	url = https://github.com/FreeRTOS/backoffAlgorithm.git
+[submodule "libraries/aws/ota"]
```

Comment:
We were going to change this to `ota-for-aws-iot-embedded-sdk` right?

Reply:
yes I'll update